explainer.md

```diff
@@ -380,19 +378,20 @@ Calls to `subscribe()` and `unsubscribe()` are idempotent.
 
 ```javascript
 self.addEventListener('activate', (event) => {
-  // Snapshot current state of subscriptions.
-  const subscriptions = await self.registration.cookies.getSubscriptions();
-
-  // Clear any existing subscriptions.
-  for (const subscription of subscriptions)
-    await self.registration.cookies.unsubscribe(subscription);
-
-  await self.registration.cookies.subscribe([
-    {
-      name: 'session',  // Get change events for session-related cookies.
-      matchType: 'starts-with',  // Matches session_id, session-id, etc.
-    }
-  ]);
+  event.waitUntil(async () =>
```

Comment:
We need to add the `waitUntil` call in the spec example in https://wicg.github.io/cookie-store/#intro-monitor too.

Reply:
Done.